modin/numpy/arr.py

```diff
@@ -485,71 +505,88 @@ def min(
         if self._ndim == 1:
             if axis == 1:
                 raise numpy.AxisError(1, 1)
-            result = self._query_compiler.min(axis=0)
+            result = self._query_compiler
+            if isinstance(where, array):
+                result = where.where(self, initial)._query_compiler
+            result = result.min(axis=0)
             if keepdims:
-                if initial is not None and result.lt(initial):
+                if initial is not None and result.lt(initial).any():
```

Comment:
```suggestion
                if initial is not None and result.gt(initial).any():
```
I wonder why we `.lt()` both in `.max()` and in `.min()`, feels like it should be `.gt()` here?..

Reply:
Good catch, thank you!